nebula_rust/src/value/data_set.rs

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+pub trait DataSet {
```

Comment:
Add ExcutionResponse wrapper

Reply:
It's data set extension not related to the response.